src/pages/index.js

```diff
@@ -27,21 +31,37 @@ import {
 
 const currentUserInfo = new UserInfo(
   selectors.profileTitle,
-  selectors.profileDescription
+  selectors.profileDescription,
+  selectors.profileAvatarContainer,
+  selectors.profileAvatarEdit,
+  selectors.avatarImage
 );
 
 const previewModal = new PopupWithImage(selectors.previewModal);
 
 const addCard = new PopupWithForm(
```

Comment:
`addCard`


Please, add word `Popup` at the end of any variable for popups


`addCardPopup`